src/Microsoft.SqlTools.ServiceLayer/QueryExecution/DataStorage/SaveAsExcelFileStreamWriterHelper.cs

```diff
@@ -0,0 +1,688 @@
+﻿
```

Comment:
One thing we wanted double check on, if this code was written by you, we'll need the MIT license header like in SaveAsExcelFileStreamWriter.cs. If the code wasn't written by you (it came from a sample or is a copy of a library), we'll need to make sure we list the source and the license it was from in here.
Sorry for another change, but it's one of the licensing hurdles we need to make sure everything is legal.